### PR TITLE
Changed bomb_enemy.py in directory matrix

### PR DESCRIPTION
**Pull request by christianbender**
I maked the code more efficient. Put in some comments and added a testsuite (unittest) for the project.

### CODE REVIEW

File: matrix/bomb_enemy.py [hunk 1]
```diff
@@ -45,18 +51,44 @@ def row_kills(grid, i, j):
 # calculate killed enemies for  column j from row i
 def col_kills(grid, i, j):
     num = 0
-    while i < len(grid) and grid[i][j] != 'W':
+    # more efficient. calls the function only once!
```
Comment (keon) [inline matrix/bomb_enemy.py]: this comment should be in the commit message, not in the file

**Comment by christianbender** [comment]
@keon  I have removed the specified comment.

File: matrix/bomb_enemy.py [hunk 2]
```diff
@@ -15,28 +15,34 @@
...
+                # updates the variable
                 max_killed = max(max_killed, row_e + col_e[j])
 
     return max_killed
 
 # calculate killed enemies for row i from column j
 def row_kills(grid, i, j):
     num = 0
-    while j < len(grid[0]) and grid[i][j] != 'W':
+    # more efficient. calls the function only once!
```
Comment (keon) [inline matrix/bomb_enemy.py]: this one too plz

**Comment by christianbender** [comment]
@keon done.

**Pull request merged**